addons/test_website/tests/test_qweb.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from lxml import etree
+import re
+
+from odoo import tools
+from odoo.addons.base.tests.common import TransactionCaseWithUserDemo
+from odoo.modules.module import get_module_resource
+
+
+class TestQweb(TransactionCaseWithUserDemo):
```

Comment:
Those should have been `@tagged('-at_install', 'post_install')` in the first place IMO, but let's merge it like this